e2e/cypress/integration/messaging/direct_messages_spec.js

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @messaging
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+import users from '../../fixtures/users.json';
+
+const userActive = users['user-1'];
```

Comment:
Instead of using the one from hard-coded users.json, you may use the "user" generated from `cy.apiInitSetup()`.